lib/sycamore/sycamore/transforms/merge_elements.py

```diff
@@ -429,12 +430,13 @@ class TableMerger(ElementMerger):
 
             llm = OpenAI(OpenAIModels.GPT_4O, api_key = '')
 
-            prompt = "Analyze two CSV tables that may be parts of a single table split across pages. Determine if the second table\
-                      is a continuation of the first with 100% certainty. Check either of the following:\
-            1. Column headers: Must be near identical in terms of text(the ordering/text may contain minor errors because of OCR quality)\
-               in both tables. If the headers are almost the same check the number of columns, they should be roughly the same.\
-            2. Missing headers: If the header/columns in the second table are missing, then the first row in the second table should logically\
-               be in continutaion of the last row in the first table.\
+            prompt = "Analyze two CSV tables that may be parts of a single table split across pages. Determine\
+            if the second table is a continuation of the first with 100% certainty. Check either of the following:\
+            1. Column headers: Must be near identical in terms of text(the ordering/text may contain minor errors \
+            because of OCR quality) in both tables. If the headers are almost the same check the number of columns,\
+                 they should be roughly the same.\
+            2. Missing headers: If the header/columns in the second table are missing, then the first row in the
```

Comment:
Is the spacing gonna do the right thing here? I think you'll get "...the same.2. Missing..." but you want "...the same. 2. Missing headers" or something. tbh I'm not sure how python escapes whitespace tho so I might be wrong

Reply:
done